app/src/main/java/com/google/samples/apps/sunflower/utilities/SingleLiveEvent.java

```diff
@@ -0,0 +1,75 @@
+/*
```

Comment:
This class looks fine, I can accept, however, this `SingleLiveData`  is for this case, maybe others, only when the `observer` single, that means, more than one `listener`s is problem.

There's a better [solution](https://medium.com/google-developers/livedata-with-snackbar-navigation-and-other-events-the-singleliveevent-case-ac2622673150). Nice2have you can try and learn some.

Reply:
@XinyueZ Good solution, I consider maybe use it later in new PR about improve SingleLiveData.